interface-definitions/vpn_ipsec.xml.in

```diff
@@ -701,6 +701,52 @@
                   <valueless/>
                 </properties>
               </leafNode>
+              <node name="retransmission">
+                <properties>
+                  <help>IPsec retrasmition settings</help>
+                </properties>
+                <children>
+                  <leafNode name="tries">
+                    <properties>
+                      <help>Number of retransmissions to send before giving up</help>
+                      <valueHelp>
+                        <format>u32:1-100</format>
+                        <description>Number of retransmissions to send before giving up</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-100"/>
+                      </constraint>
+                    </properties>
+                    <defaultValue>5</defaultValue>
+                  </leafNode>
+                  <leafNode name="base">
+                    <properties>
+                      <help>Base of exponential backoff</help>
+                      <valueHelp>
+                        <format>&lt;0.001-1000&gt;</format>
```

Comment:
Where are those values from? I see no min/max at https://docs.strongswan.org/docs/latest/config/retransmission.html

and would rather use min 1.9 max 5.0 - otherwise the resulting timeouts will be really long

Reply:
I have changed to min 1.0 max 5.0